core/src/main/java/org/infinispan/commands/AbstractTopologyAffectedCommand.java

```diff
@@ -8,9 +10,15 @@
  */
 public abstract class AbstractTopologyAffectedCommand extends AbstractFlagAffectedCommand implements TopologyAffectedCommand {
 
-   private int topologyId = -1;
+   protected int topologyId;
+
+   protected AbstractTopologyAffectedCommand(long flags, int topologyId) {
+      super(flags);
+      this.topologyId = topologyId;
+   }
 
    @Override
+   @ProtoField(number = 2, defaultValue = "-1")
```

Comment:
why setting the default value to -1 in numeric fields and not use the java/protostream default of zero? 
The question is not related to this particular field, but I've seen this almost everywhere.

Reply:
Good question. When I originally started working on this we only had PROTO2, so the defaultValue was always required. I *think* the reason I went with -1 was to aid with debugging, but I really can't be sure. I'll remove  the `defaultValue` definitions where a 0 value makes sense.

Reply:
I have added a commit that removes all redundant `defaultValue` and `number/value = ` from `@ProtoField` definitions. I have deliberately only applied the change to `@ProtoField` adapters updated as part of this PR as I didn't want to increase the diff further. If we want to modify any existing definitions, I think it should happen in a subsequent PR.

https://github.com/infinispan/infinispan/pull/13316/commits/a62e5794cadf325ac0ca1aeb43fac9dbf573e590